com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+            SerializedProperty lensFlareProp = property.FindPropertyRelative("lensFlareTexture");
+            SerializedProperty tintProp = property.FindPropertyRelative("tint");
+            SerializedProperty blendModeProp = property.FindPropertyRelative("blendMode");
+            SerializedProperty countProp = property.FindPropertyRelative("count");
+            SerializedProperty allowMultipleElementProp = property.FindPropertyRelative("allowMultipleElement");
+            SerializedProperty rotationProp = property.FindPropertyRelative("rotation");
+            SerializedProperty speedProp = property.FindPropertyRelative("speed");
+            SerializedProperty autoRotateProp = property.FindPropertyRelative("autoRotate");
+            SerializedProperty preserveAspectRatioProp = property.FindPropertyRelative("preserveAspectRatio");
+            SerializedProperty modulateByLightColor = property.FindPropertyRelative("modulateByLightColor");
+            SerializedProperty isFoldOpenedProp = property.FindPropertyRelative("isFoldOpened");
+            SerializedProperty flareTypeProp = property.FindPropertyRelative("flareType");
+
+            SerializedProperty uniformScaleProp = property.FindPropertyRelative("uniformScale");
+            SerializedProperty sizeXYProp = property.FindPropertyRelative("sizeXY");
+
+            //
+            SerializedProperty distributionProp = property.FindPropertyRelative("distribution");
+
+            SerializedProperty lengthSpreadProp = property.FindPropertyRelative("lengthSpread");
+            SerializedProperty colorGradientProp = property.FindPropertyRelative("colorGradient");
+            SerializedProperty positionCurveProp = property.FindPropertyRelative("positionCurve");
+            SerializedProperty scaleCurveProp = property.FindPropertyRelative("scaleCurve");
+
+            // Random
+            SerializedProperty seedProp = property.FindPropertyRelative("seed");
+            SerializedProperty intensityVariationProp = property.FindPropertyRelative("intensityVariation");
+            SerializedProperty positionVariationProp = property.FindPropertyRelative("positionVariation");
+            SerializedProperty scaleVariationProp = property.FindPropertyRelative("scaleVariation");
+            SerializedProperty sizeVariationProp = property.FindPropertyRelative("sizeVariation");
+            SerializedProperty rotationVariationProp = property.FindPropertyRelative("rotationVariation");
+
+            // Distortion
+            SerializedProperty enableDistortionProp = property.FindPropertyRelative("enableRadialDistortion");
+            SerializedProperty targetSizeDistortionProp = property.FindPropertyRelative("targetSizeDistortion");
+            SerializedProperty distortionCurveProp = property.FindPropertyRelative("distortionCurve");
+            SerializedProperty distortionRelativeToCenterProp = property.FindPropertyRelative("distortionRelativeToCenter");
```

Comment:
All of this should be constructed once and stored, not done at each OnGUI(). Finding property have a small cost that can be cached

Reply:
Currently they are relative property, relative to property. Property is only accessible from OnGUI();

Reply:
Oh right 😞 Sorry I was thinking it is an Editor instead a PropertyDrawer. Yes you have no choice here :/